src/guides/aaa_general/night-action-order.mdx

```diff
@@ -0,0 +1,34 @@
+---
+name: Basics
+menu: Night Action Order
+route: /night-action-order
+---
+
+# Order of Night Actions
+
+The different abilities roles have at night may influence each other. How they do is determined by the order in which the night actions are defined to take place. The order of night actions is:
+
+1. Redirects (Succubus)
+1. Roleblocks (Direwolf, Courtesan, Manacles of Spite)
+1. Protection visits (Protector, Huntsman, Shaman, Armour, Heavy Shield)
+1. Most visits (Too many to list: all visits not belonging in another category, including items)
+1. Item thefts from abilities and potions
+1. Kills (Killing werewolf (Werewolf/Alphawolf/Shapeshifter), Vampire, Witch, Militia), including resulting item thefts
+1. Report visits to Stalkers, Harlots, Familiar
+1. Passing of items
+1. Swap Identities (Djinn, Shapeshifter)
+1. Report kills to village
+1. Report revives to village
```

Comment:
all marked as 1.

Reply:
That's a Markdown trick - you can mark everything in the ordinal list as 1 and it will create the right numbering for you.

Reply:
Technically markdown will do the right thing here, but it would be nicer to be numbered correctly in the source

Reply:
TIL...

Reply:
I prefer it as well, as it keeps the Git history cleaner - if someone were to add an entry, they would not need to change the following lines and influence the `git blame` output.